# A name for this grasp
string id

# The position of the end-effector for the grasp.  This is the pose of
# the "parent_link" of the end-effector, not actually the pose of any
# link in the end-effector.  Typically this would be the pose of the
# most distal wrist link before the hand (end-effector) links began.
geometry_msgs/PoseStamped grasp_pose

# the maximum contact force to use while grasping (<=0 to disable)
float32 max_contact_force

# the maximum velocity to use while grasping (<=0 to disable)
float32 max_contact_velocity

# the maximum width to use while grasping (<=0 to disable)
float32 max_contact_width

# The position of the end-effector for the release.  This is the pose of
# the "parent_link" of the end-effector, not actually the pose of any
# link in the end-effector.  Typically this would be the pose of the
# most distal wrist link before the hand (end-effector) links began.
#geometry_msgs/PoseStamped release_pose
---
bool grasp_result